apps/examples/wc-nuxt3/package.json

```diff
@@ -6,16 +6,19 @@
     "build:examples": "yarn build",
     "dev": "nuxt dev",
     "generate": "nuxt generate",
+    "start": "nuxt start",
     "postinstall": "nuxt prepare"
   },
   "dependencies": {
-    "@justeattakeaway/pie-button": "^0.10.1",
-    "@nuxt/vite-builder": "3.0.0",
-    "vue": "3.2.25"
```

Comment:
Do we not need this dependency anymore?

Reply:
While I was trying to figure the reason for the [Hydration node mismatch](https://github.com/prashantpalikhe/nuxt-ssr-lit/issues/73) warnings I setup a Next 3 project from scratch. During the debug process I experimented removing and adding settings and dependencies. Ended up finding that some were really unnecessary.

https://github.com/fernandofranca/nuxt-3-lit-ssr